dg/lib/ibmcloud/__init__.py

```diff
@@ -1,5 +1,37 @@
+import subprocess
+
 from typing import Final
 
+import dg.utils.process
+
+from dg.config import data_gate_configuration_manager
```

Comment:
If the only method we need from `data_gate_configuration_manager`is `get_ibmcloud_cli_path`we can only import this function

Reply:
I do not think it is possible to import a method of an object within a package:

```
from dg.config.data_gate_configuration_manager import get_ibmcloud_cli_path
```

→ `Import "dg.config.data_gate_configuration_manager" could not be resolved `

Or did you mean something else?